riscv/src/continuations/bootloader.rs

```diff
@@ -165,11 +181,12 @@ submachine_init:
 // START OF BOOTLOADER
 
 // During the execution of the bootloader, registers are used as follows:
+// (NB. `SYSCALL_REGISTERS` are mapped into the `P*` registers)
```

Comment:
didn't we remove the P registers?

Reply:
ah I see now

Reply:
will change `registers` for `variables` in the comment here